lib/type/string.ex

```diff
@@ -36,6 +37,25 @@ defmodule Litmus.Type.String do
     end
   end
 
+  @spec convert(t, binary, map) :: {:ok, map}
+  defp convert(%__MODULE__{}, field, params) do
+    cond do
+      !Map.has_key?(params, field) ->
+        {:ok, params}
+
+      is_binary(params[field]) ->
+        {:ok, params}
+
+      is_number(params[field]) or is_boolean(params[field]) ->
+        modified_value = to_string(params[field])
+        modified_params = Map.put(params, field, modified_value)
+        {:ok, modified_params}
```

Comment:
I think we could condense the `to_string` and `Map.put` steps into a single step using [`Map.update!/3`](https://hexdocs.pm/elixir/Map.html#update!/3).

So this would look something like
```elixir
modified_params = Map.update!(params, field, &to_string/1)
{:ok, modified_params}
```

I would even be fine with making this into a single line:

```elixir
{:ok, Map.update!(params, field, &to_string/1)}
```